File: Data Structure/Array/Array Data Structure Guide/Basic Problems on Array/Largest element in an Array/method3.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int largest(vector<int>& arr) {
    return *max_element(arr.begin(), arr.end());
}

int main() {
    vector<int> arr = {10, 324, 45, 90, 9808};
    cout << largest(arr);
    return 0;
}

File: Data Structure/Array/Array Data Structure Guide/Basic Problems on Array/Largest element in an Array/method3.java
import java.io.*;
import java.util.*;

class Main {

    static int largest(int[] arr) {
        Arrays.sort(arr);
        return arr[arr.length - 1];
    }

    static public void main(String[] args) {
        int[] arr = { 10, 324, 45, 90, 9808 };
        System.out.println(largest(arr));
    }
}